utils/src/commonMain/kotlin/com/badoo/reaktive/utils/lock/LockExt.kt

```diff
@@ -15,4 +15,4 @@ inline fun <T> Lock.use(block: (Lock) -> T): T =
     } finally {
         destroy()
     }
-    
\ No newline at end of file
+        
```

Comment:
Detekt failed because of this spaces. Keep one empty line without spaces.

Reply:
Also you can check it locally before push by invoking `detekt` Gradle task.

Reply:
Thank you @CherryPerry , I have done this 👍